Content.Client/Playtime/ClientsidePlaytimeTrackingManager.cs

```diff
@@ -105,4 +123,75 @@ private void OnPlayerDetached(EntityUid entity)
 
         _configurationManager.SaveToFile(); // We don't like that we have to save the entire config just to store playtime stats '^'
     }
+
+    #region Moffstation - Hourly Playtime Notice
+
+    /// <summary>
+    /// Schedules the next hourly playtime notice at the top of the next hour.
+    /// </summary>
+    private void ScheduleNextHourlyNotice()
+    {
+        try
+        {
+            _hourlyNoticeCts?.Cancel();
+            _hourlyNoticeCts?.Dispose();
+        }
+        catch
+        {
+            // l plus ratio
+        }
+
+        _hourlyNoticeCts = new CancellationTokenSource();
+
+        var now = DateTime.Now;
+        var nextHour = new DateTime(now.Year, now.Month, now.Day, now.Hour, 0, 0, now.Kind).AddHours(1);
+        var delay = nextHour - now;
+        var ms = Math.Max(0, (int)Math.Ceiling(delay.TotalMilliseconds));
+
+        RobustTimer.Spawn(ms,
```

Comment:
```suggestion

        RobustTimer.Spawn(new[] { TimeSpan.Zero, delay }.Max(),
```
imagine your stdlib not having a
```csharp
public static T Max<T>(T left, T right) where T : IComparable<T>
```